generators/new_collateral/config/parser.go

```diff
@@ -0,0 +1,85 @@
+package config
+
+import (
+	"errors"
+
+	"github.com/BurntSushi/toml"
+	"github.com/makerdao/vdb-mcd-transformers/generators/new_collateral/helpers"
+	"github.com/makerdao/vdb-mcd-transformers/generators/new_collateral/types"
+	"github.com/mitchellh/mapstructure"
+)
+
+type IParse interface {
+	ParseCurrentConfig(configFilePath, configFileName string) (types.TransformersConfig, error)
+}
+type Parser struct{}
```

Comment:
Really minor, but I wonder if we might want to name the interface `Parser` and then not export the `struct{}` (e.g. make it `parser`) and then use a constructor to return that private implementation

Reply:
That would be more go-ish wouldn't it?

Reply:
Nice call. 

I was a bit on the fence on wether `parser{}` is really necessary at all since we're not setting any state on it. I decided to put `ParseCurrentConfig` on a parser receiver, so I could create an interface, and use that for testing. But up for other ideas too!

Reply:
addressed in 0018720bd0c7d45029d0e522e0f9e015f202fee8

Reply:
addressed in 0018720bd0c7d45029d0e522e0f9e015f202fee8